frontend/src/app/spindle/spindle-about/spindle-about.component.html

```diff
@@ -0,0 +1,73 @@
+<div class="content is-size-5">
```

Comment:
Do we need the `is-size-5`? It makes the text on this page really stand out from the rest of the website, where the base text is smaller.